hvplot/converter.py

```diff
@@ -454,11 +492,17 @@ class HoloViewsConverter:
         'min_width',
         'frame_height',
         'frame_width',
-        'aspect',
-        'data_aspect',
         'fontscale',
         'bgcolor',
         'subcoordinate_y',
+        'clim',
+        'flip_xaxis',
+        'flip_yaxis',
+        'tools',
+        'hover',
```

Comment:
I don't know if the hover options really fit within axis options?

Reply:
In general, I wasn't really sure where to put all the `hover_` related things...